content/id/docs/tutorials/kubernetes-basics/update/update-intro.html

```diff
@@ -0,0 +1,135 @@
+---
+title: Melakukan Pembaruan Bertahap
+weight: 10
+---
+
+<!DOCTYPE html>
+
+<html lang="id">
+
+<body>
+
+    <link href="/docs/tutorials/kubernetes-basics/public/css/styles.css" rel="stylesheet">
+
+
+<div class="layout" id="top">
+
+    <main class="content">
+
+        <div class="row">
+
+     <div class="col-md-8">
+          <h3>Tujuan</h3>
+                <ul>
+                    <li>Melakukan pembaruan bertahap (<i>rolling update</i>) menggunakan kubectl.</li>
+                </ul>
+            </div>
+
+            <div class="col-md-8">
+            <h3>Memperbarui suatu aplikasi</h3>
+
+            <p>Pengguna mengharapkan aplikasi tersedia sepanjang waktu dan pengembang bisa jadi diharapkan untuk men-<i>deploy</i> versi terbaru dari aplikasi tersebut beberapa kali dalam suatu waktu. Di Kubernetes hal ini dilakukan melalui pembaruan bertahap. <b>Pembaruan bertahap</b> memungkinkan pembaruan Deployment terjadi tanpa waktu henti dengan memperbarui instans Pod secara bertahap dengan versi yang baru. Pod yang baru akan dijadwalkan pada Node dengan sumber daya yang tersedia.</p>
+
+            <p>Dalam modul sebelumnya kita telah menskalakan aplikasi kita untuk menjalankan beberapa instans. Hal ini dibutuhkan untuk menjalankan pembaruan tanpa mempengaruhi ketersediaan aplikasi. Secara <i>default</i>, jumlah maksimum dari Pod lama yang akan digantikan (menjadi tidak tersedia) selama pembaruan dan jumlah maksimum Pod baru yang dapat dibuat adalah satu. Kedua opsi ini dapat dikonfigurasi dengan angka atau persentase (dari Pod). Di Kubernetes, setiap pembaruan diberi versi masing-masing dan suatu pembaruan Deployment dapat dikembalikan ke versi sebelumnya (yang stabil).</p>
```

Comment:
“menskalakan” seems not official, please use “penyekalaan” https://kbbi.kemdikbud.go.id/entri/penyekalaan
use “bawaan” untuk “default”
```suggestion
            <p>Dalam modul sebelumnya kita telah melakukan penyekalaan aplikasi kita untuk menjalankan beberapa instans. Hal ini dibutuhkan untuk menjalankan pembaruan tanpa mempengaruhi ketersediaan aplikasi. Secara bawaan (<i>default</i>), jumlah maksimum dari Pod lama yang akan digantikan (menjadi tidak tersedia) selama pembaruan dan jumlah maksimum Pod baru yang dapat dibuat adalah satu. Kedua opsi ini dapat dikonfigurasi dengan angka atau persentase (dari Pod). Di Kubernetes, setiap pembaruan diberi versi masing-masing dan suatu pembaruan Deployment dapat dikembalikan ke versi sebelumnya (yang stabil).</p>
```

Reply:
Done in 11ec389